src/query/plan/operator.cpp

```diff
@@ -1677,6 +1701,10 @@ class SingleSourceShortestPathCursor : public query::plan::Cursor {
 namespace {
 
 void CheckWeightType(TypedValue current_weight, utils::MemoryResource *memory) {
+  if (current_weight.IsNull()) {
+    return;
+  }
```

Comment:
We need this, because weight at the beginning of path calculation when there is no edge, could be Null.
For example, we have a query: `MATCH (n {a:'0'})-[le *allShortest 1 (e, n | e.w ) w]->(m) RETURN m.a`
In this query when we calculate beginning weight we put to frame begin node and NULL as edge. So it's possible to weight to be null. During expansion weight will be updated.